subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -847,23 +841,14 @@ void ull_conn_setup(memq_link_t *rx_link, struct node_rx_hdr *rx)
 	 */
 	hdr = HDR_LLL2ULL(ftr->param);
 	if (ull_ref_get(hdr)) {
-		uint32_t ret;
-
+		/* Setup connection in ULL disabled callback,
+		 * pass the node rx as disabled callback parameter.
+		 */
 		LL_ASSERT(!hdr->disabled_cb);
-		hdr->disabled_param = mfy.param;
+		hdr->disabled_param = rx;
 		hdr->disabled_cb = conn_setup_adv_scan_disabled_cb;
-
-		mfy.fp = lll_disable;
```

Comment:
The original code calls `lll_disable` via mayfly, while the new code does not call `lll_disable`. It is not obvious to my why that call is no longer necessary. Can you explain?

Reply:
Original call passed `mfy.param = rx` (incorrectly) when calling `lll_disable` which in theory is redundant, as `lll_disable` will not find a LLL context pointed to be `rx`. The connection was still established as ULL reference count reached zero due to `lll_done` being invoked in LLL context of advertiser  and scanner.

Reply:
So the rationale is that because `lll_disable` was not called correctly before (had no consequence), and it still works without that, then we skip calling it altogether? Not sure it makes sense to me... `lll_disable` performs a whole lot of shutting down/aborting and resetting stuff in vendor LLL. Please elaborate.

Reply:
I can add that our simulation tests pass with this PR, so with a background story on addition and removal of `lll_disable`, I can approve.

Reply:
‘lll_disable’ call got redundant when flags were added (necessary) to gracefully handle race conditions when establishing connection. the implementation was incorrect anyway, doing nothing to abort any LLL in the pipeline due to incorrect context pointer passed.

Please provide your approval if there is no concerns on inspection and if internal tests pass.